tests/apollo/util/bft.py

```diff
@@ -629,6 +631,24 @@ def start_replica_cmd(self, replica_id):
                 cmd.append(keys_path)
             return cmd
 
+    def check_error_logs(self):
+        """
+        Checking ERROR/FATAL logs in replica logs and writing in a file.
+        If the file exists for any testcase, warning is displayed in CI
+        """
+        with log.start_action(action_type="replica_log_scanner") as action:
+            cmd = ['grep', '-R', 'ERROR\|FATAL', '--exclude=err_logs']
```

Comment:
Please rename the filename err_logs for better understandability

Reply:
renamed to ReplicaErrorLogs.txt